README.md

```diff
@@ -203,8 +254,110 @@ List of available snippets:
+| &emsp;
+| **PERIOD**
+| period.accounts | Add the method to return a collection of all accounts with bookings on this period |
+| period.account_mapping_list.name | Add the method to return the name of the mapping list used for this period |
+| period.account_mapping_list.id | Add the method to return the id of the mapping list on firm level |
+| period.account_mapping_list.marketplace_template_id | Add the method to return the id of the mapping list on marketplace |
+| period.adjustments | Add the method to return all the adjustments for the period |
+| period.bookyear_index | Add the method to return the  index number of the current book year as an integer. The first book year equals 1 |
+| period.calendar_years | Add the method to add an array of all calendar years in the fiscal year. The information for each calendar year is:<br>start_date, end_date, amount_of_days for the number of days the bookyear has in the calendar year, and<br> amount_of_days_in_full_year for the total number of days in the calendar year |
+| period.custom | Add method to attach custom information to a period. This is done automatically in reconciliations |
+| period.directors | Add method to return people drop of *all* individuals who are directors |
+| period.end_date | Add method to return the date this period ends |
+| period.exists | Add method to return true when the period exists in the Silverfin database |
+| period.fiscal_year | Add method to return the fiscal year of this period |
+| period.is_first_year | Add method to return true if the period is in the first book year of this client file |
+| period.month_end_dates | Add method to return an array of all the end dates of the calendar months in this fiscal year |
+| period.name | Add method to return the name of the period |
+| period.people | Add method to return the people drop of *all* people attached to the period (typically copied from general<br> company level) |
+| period.reconciliations | Add method to return a collection of all reconciliations for this period. You can ask for a specific reconciliation<br> by adding the handle. I.e. period.reconciliations.the_handle. |
+| period.reports | Add method to return a collection of all reports for this period. You can ask for a specific report by adding the<br> handle. I.e. period.reports.the_handle |
+| period.shareholders | Add method to return the people drop of *all* individuals who are shareholders |
+| period.start_date | Add method to return the date this period starts |
+| period.year_end | Add method to return the period at the end of the fiscal year this period is in |
+| period.year_end_date | Add method to return the date of the end of the fiscal year of this period |
+| period.year_start_date | Add method to return the date of the start of the fiscal year of this period |
+| period.minus_1p,<br>period.minus_2p,<br>period.minus_3p,<br>... | Add method to return the period drop of the current period minus the amount of periods defined |
```

Comment:
I think the snippet will return something more like `period.minus_xp`? 
We could also add one that's already a static `period.minus_1p`, `period.minus_1y`, `period.minus_1y.year_end`, cause those are the most used ones.

Reply:
Agreed and updated